lib/algos/list/delimiterMaster.js

```diff
@@ -33,6 +33,8 @@ class DelimiterMaster extends Delimiter {
         this.prvKey = undefined;
         this.prvPHDKey = undefined;
         this.inReplayPrefix = false;
+        this.prefixKeySeen = false;
+        this.delimEnd = this.prefix && this.prefix.endsWith(this.delimiter);
```

Comment:
Let's be explicit in the variable name

```suggestion
        this.prefixEndsWithDelim = this.prefix && this.prefix.endsWith(this.delimiter);
```